src/tests/end-to-end/common/screenshot-on-error.ts

```diff
@@ -1,26 +1,25 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { createDefaultPromiseFactory } from 'common/promises/promise-factory';
-import * as Puppeteer from 'puppeteer';
 import { DEFAULT_SCREENSHOT_TIMEOUT_MS } from 'tests/end-to-end/common/timeouts';
 import { prepareTestResultFilePath } from './prepare-test-result-file-path';
 
 const promiseFactory = createDefaultPromiseFactory();
 
-async function takeScreenshot(pageInstance: Puppeteer.Page): Promise<void> {
+async function takeScreenshot(saveScreenshot: (path) => Promise<Buffer>): Promise<void> {
```

Comment:
Does this make sense, or should the framework-specific screenshot mechanism be abstracted away differently?